cpp/CMakeLists.txt

```diff
@@ -82,8 +85,42 @@ set(GPU_ARCHS "" CACHE STRING
     Pass 'ALL' if you want to compile for all supported GPU architectures.
     Empty string means to auto-detect the GPUs on the current system")
 
-set(CMAKE_IGNORE_PATH "${CMAKE_INSTALL_DIR}/lib" CACHE STRING
-    "Ignore any libs added implicitly from the CMAKE_INSTALL_DIR")
+
+##############################################################################
+# - Conda environment detection ----------------------------------------------
+
+# conda-build conda package building detection
+if("$ENV{CONDA_BUILD}" STREQUAL "1")
```

Comment:
This env variable doesn't seem to be documented? Is this some known variable?

Reply:
Its an environment variable that is used only when using `conda-build` to create the conda packages https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#id2

Reply:
Ah.. ok. Thanks Dante. Makes sense now.